packages/gatsby-theme-newrelic/gatsby-node.js

```diff
@@ -401,8 +419,8 @@ const createRelatedResources = async (
   const { createNode, createParentChildLink } = actions;
 
   if (
-    node.internal.type !== 'Mdx' ||
-    node.fileAbsolutePath.includes(ANNOUNCEMENTS_DIRECTORY)
+    (node.internal.type !== 'Mdx' && node.internal.type !== 'Quickstarts') ||
```

Comment:
Totally a style thing, but a slightly more compact version that might be easier to add onto later could be this:

```suggestion
    !['Mdx', 'Quickstarts'].includes(node.internal.type) ||
```

Reply:
If we wanted to be very fancy and abstract it, we could add the option to supply this via theme options. Something like `releatedResourceNodeTypes` (but with a better name). That way we wouldn't have to hard-code a reference to quickstarts here and we could use that for the `createResolvers` bit of this code block too.

```js
const RELATED_RESOURCE_RESOLVER = { /* common settings */ };

createResolvers({
  // ...
  ...themeOptions.relatedResourceNodeTypes.reduce((acc, type) => ({
    ...acc,
    [type]: RELATED_RESOURCE_RESOLVER,
  }), {}),
  // ...
});
```

```js
!['Mdx', ...themeOptions.relatedResourceNodeTypes].includes(node.internal.type) ||
```

To be honest, I'm not sure if that helps, or if it makes it way too complicated. Just trying to explore ways we can keep this abstract since this is pretty specific to only one of our sites.

Reply:
i AM a fan of fancy things but i think for this PR i'm going to hold off on that, though i will implement the array check because it's nicer to look at!